src/Stratis.Bitcoin.Tests/P2P/PeerConnectorTests.cs

```diff
@@ -27,7 +28,7 @@ public void PeerConnectorAddNode_FindPeerToConnectTo_Returns_AddNodePeers()
                 ConnectionManager = new Configuration.Settings.ConnectionManagerSettings()
             };
             nodeSettings.ConnectionManager.AddNode.Add(networkAddressAddNode.Endpoint);
-            var connector = new PeerConnectorAddNode(nodeSettings, peerAddressManager);
+            var connector = new PeerConnectorAddNode(new DateTimeProvider(), nodeSettings, peerAddressManager);
```

Comment:
everywhere

Reply:
OK fixed this in all the applicable tests.